convert2rhel/unit_tests/systeminfo_test.py

```diff
@@ -163,8 +162,8 @@ def test_modified_rpm_files_diff_with_differences_after_conversion(self):
     @unit_tests.mock(utils, "run_subprocess", RunSubprocessMocked(("rpmva\n", 0)))
     def test_generate_rpm_va(self):
         # Check that rpm -Va is executed (default) and stored into the specific file.
+        tool_opts.no_rpm_va = False
```

Comment:
I had to include this line in this test, otherwise, it would still be failing after adding the tests in the `subscription_test.py`. Probably we have some leaks when using the `pretend_os` fixture.